source/elements/oneTBB/source/nested-depr-interfaces.rst

```diff
@@ -0,0 +1,12 @@
+.. SPDX-FileCopyrightText: 2021 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+=============================
+ oneTBB Deprecated Interfaces

```

Comment:
Do we really want to contain deprecated interfaces documentation in the specification? Shouldn't we just remove them from spec and add them to reference as deprecated?

Reply:
I think it is not the right approach. If we just remove the functionality from the spec - some vendor (for example not Intel oneTBB) can just remove this functionality from the implementation (or not implement it at all). But all of the functionality in oneAPI should provide backward compatibility guarantees - so such a thing will brake this logic.  C++ Standard follows the same scheme, e.g. `std::auto_ptr` was deprecated in C++11 and removed in C++17, so each vendor which implements C++11 and C++14 should keep the implementation of `std::auto_ptr` to simplify the migration for users.

Reply:
Thanks for the exploration, agree.